examples/llamaindex/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        eval_df = eval_df.style.set_properties(
</pre><p> </p><p>Could you use the colab formatter instead?</p><p> </p>

<pre class="ql-syntax" spellcheck="false">from google import colab
colab.data_table.enable_dataframe_formatter()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>